plugins/zfs/zfs.go

```diff
@@ -32,7 +37,7 @@ func (z *Zfs) Description() string {
 	return "Read metrics of ZFS from arcstats, zfetchstats and vdev_cache_stats"
 }
 
-func getTags(kstatPath string) map[string]string {
```

Comment:
I know it isn't ideal, I wanted to avoid doing the directory scan a second time for each Gather call.